client/src/views/users/GreetUserModal.vue

```diff
@@ -0,0 +1,442 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS -->
+
+<template>
+  <ion-page>
+    <ms-wizard-stepper
+      v-show="pageStep > GreetUserStep.WaitForGuest && pageStep < GreetUserStep.Summary"
+      :current-index="getStepperIndex()"
+      :titles="[
+        'Code admin',
+        'Code invité',
+        'Contact details',
```

Comment:
Why is there plain text here?
Should use locales.
French and English are mixed.